### PR TITLE
Added Default prefix for MacOS on ARM in Uninstaller script

### PR DESCRIPTION
**Pull request by jiteshpubreja**
Corrected Uninstall script for detecting location of homebrew automatically in M1 Macs.

Tried Multiple times and confirmed to be working.

### CODE REVIEW

File: uninstall.sh [hunk 1]
```diff
@@ -42,14 +42,19 @@ opt_skip_cache_and_logs=""
...
+unm=$(uname -m)
 case "$un" in
   Linux)
     ostype=linux
     homebrew_prefix_default=/home/linuxbrew/.linuxbrew
   ;;
   Darwin)
     ostype=macos
-    homebrew_prefix_default=/usr/local
+    if [[ "$unm" == "arm64" ]]; then
```
Comment (MikeMcQuaid) [inline uninstall.sh:53]:
```suggestion
    if [[ "$(uname -m)" == "arm64" ]]; then
```

File: uninstall.sh [hunk 2]
```diff
@@ -42,14 +42,19 @@ opt_skip_cache_and_logs=""
 failed=false
 
 un=$(uname)
+unm=$(uname -m)
```
Comment (MikeMcQuaid) [inline uninstall.sh:45]:
```suggestion
```

**Review by MikeMcQuaid — COMMENTED** [review]
Thanks for the PR! Some minor refactoring suggestion.

File: uninstall.sh (hunk 1 above)
Comment (jiteshpubreja) [reply uninstall.sh]: Added New commit to the pr resolving above suggestions. Thank you.

File: uninstall.sh (hunk 2 above)
Comment (jiteshpubreja) [reply uninstall.sh]: Added New commit to the pr resolving above suggestions. Thank you.

**Comment by jiteshpubreja** [comment]
Hi, Thank you for your suggestions, I've updated the code to reflect that. Kindly merge the PR whenever it's convenient.

Thanks

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks so much for your first contribution! Without people like you submitting PRs we couldn't run this project. You rock, @jiteshpubreja!

**Pull request merged**